client/js/src/main/scala/zio/zmx/client/frontend/views/PanelConfigDialog.scala

```diff
@@ -111,13 +111,13 @@ object PanelConfigDialog {
         cls := "modal",
         child <-- $cfg.map { cfg =>
           div(
-            cls := "modal-box max-w-full h-5/6 mx-12 border-2 flex flex-col bg-accent-focus text-accent-content overflow-y-auto",
+            cls := "modal-box max-w-full h-5/6 mx-12 border-2 flex flex-col bg-accent-focus text-accent-content",
             div(
               cls := "border-b-2",
               span("Panel configuration")
             ),
             div(
-              cls := "flex flex-col flex-grow",
+              cls := "flex flex-col flex-grow overflow-y-auto",
```

Comment:
This makes the action buttons (`Cancel` and `Apply`) always visible, no need to scroll to the bottom for the buttons.